pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -787,32 +788,38 @@
   cs_ambassador_subhead: "If you're a high school student passionate about encouraging more young women to take computer science at your school, then we need you!"
   cs_ambassador_desc: "Every field, including medicine, entertainment, sports, education, business, fashion, and entrepreneurship is touched by computer science. Yet only 5% of high school aged young women take computer science classes. Ambassadors can help close this gender gap by encouraging their peers to take a computer science class."
   cs_ambassador_application_date: "Applications open until November 8th, 2023"
-  cs_ambassador_application_subhead: "Work with a teacher to submit your application today"
+  cs_ambassador_priority_deadline: "Priority deadline: June 28, 2024"
+  cs_ambassador_application_subhead: "Work with a teacher to submit your application today."
   cs_ambassador_application_closed_title: "Applications have now closed."
   cs_ambassador_application_closed_subhead: "Questions about future opportunities or want to learn more? Please contact [youngwomen@code.org](%{email_url})."
   cs_ambassador_application_questions: "Questions? Please contact [youngwomen@code.org](%{email_url})"
+  cs_ambassador_application_parter_program: "Not a high school student or teacher? Check out our [Partner Program.](#partner-program)"
```

Comment:
Nit: should the period be outside the link?

Reply:
<img width="357" alt="Screenshot 2024-03-19 at 12 51 17 PM" src="https://github.com/code-dot-org/code-dot-org/assets/37230822/9007c8aa-78a0-4ed9-9efa-efb565345f33">
It was inside the link in the spec, so not sure if it matters!

Reply:
Let's move it to the outside.

Reply:
Moved!